exercises/difference-of-squares/src/example/java/DifferenceOfSquaresCalculator.java

```diff
@@ -1,19 +1,19 @@
 import java.util.stream.IntStream;
 
-public final class Difference {
+public final class DifferenceOfSquaresCalculator {
 
-    public static int computeSquareOfSumTo(final int input) {
+    public int computeSquareOfSumTo(final int input) {
         final int sum = input * (input + 1) / 2;
         return (int) Math.pow(sum, 2);
     }
 
-    public static int computeSumOfSquaresTo(final int input) {
+    public int computeSumOfSquaresTo(final int input) {
         return IntStream.rangeClosed(1, input)
                 .map(i -> (int) Math.pow(i, 2))
                 .sum();
     }
 
-    public static int betweenSquareOfSumAndSumOfSquaresTo(final int input) {
+    public int betweenSquareOfSumAndSumOfSquaresTo(final int input) {
```

Comment:
Given the rename of the test class, maybe updating this to `computeDifferenceOfSquares` would be clearer and shorter now?